code/game/objects/items/inducer.dm

```diff
@@ -185,6 +185,10 @@
 		balloon_alert(user, "fully charged!")
 		return ITEM_INTERACT_FAILURE
 
+	if(iscyborg(user) && iscyborg(interacting_with))
```

Comment:
No snowflake typechecking for cyborg.

Just display the balloon alert in `/obj/item/inducer/cyborg/item_interaction`